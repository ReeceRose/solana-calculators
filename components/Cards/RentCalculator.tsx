export default function RentCalculator() {
  return (
    <>
      <div className="relative flex flex-col w-full min-w-0 mb-6 break-words border-0 rounded-lg shadow-lg dark:bg-gray-700">
        <div className="px-6 py-6 mb-0 bg-white rounded-t dark:bg-gray-900">
          <div className="flex justify-between text-center">
            <h6 className="text-xl font-bold text-blue-500">Rent Calculator</h6>
            <button
              className="px-4 py-2 mr-1 text-xs font-bold text-blue-500 uppercase transition-all duration-150 ease-linear bg-gray-100 rounded shadow outline-none dark:text-white dark:bg-gray-500 active:bg-blueGray-600 hover:shadow-md focus:outline-none"
              type="button"
            >
              Calculate
            </button>
          </div>
        </div>
        <div className="flex-auto px-4 py-10 pt-0 lg:px-10">
          <form>
            <hr className="mt-6 border-b-1 border-blueGray-300" />

            <h6 className="mt-3 mb-6 text-sm font-bold text-gray-600 uppercase dark:text-white">
              Current Price per byte-year:{' '}
              <span className="font-extrabold text-gray-700 dark:text-blue-500">
                0.00000348 SOL
              </span>
            </h6>
          </form>
        </div>
      </div>
    </>
  );
}
