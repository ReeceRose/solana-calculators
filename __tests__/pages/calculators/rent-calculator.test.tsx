import { render } from '@testing-library/react';

import RentCalculator from '../../../pages/calculators/rent-calculator';

test('calculators/rent-calculators page matches snapshot', () => {
  const { asFragment } = render(<RentCalculator />);
  expect(asFragment()).toMatchSnapshot();
});
