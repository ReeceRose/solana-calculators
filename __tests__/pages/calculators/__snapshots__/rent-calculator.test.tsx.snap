// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calculators/rent-calculators page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex-grow w-full px-4 mt-12 mb-0"
  >
    <div
      class="flex flex-wrap"
    >
      <div
        class="w-full px-4 mx-auto lg:w-8/12"
      >
        <div
          class="relative flex flex-col w-full min-w-0 mb-6 break-words border-0 rounded-lg shadow-lg dark:bg-gray-700"
        >
          <div
            class="px-6 py-6 mb-0 bg-white rounded-t dark:bg-gray-900"
          >
            <div
              class="flex justify-between text-center"
            >
              <h6
                class="text-xl font-bold text-blue-500"
              >
                Rent Calculator
              </h6>
              <button
                class="px-4 py-2 mr-1 text-xs font-bold text-blue-500 uppercase transition-all duration-150 ease-linear bg-gray-100 rounded shadow outline-none dark:text-white dark:bg-gray-500 active:bg-blueGray-600 hover:shadow-md focus:outline-none"
                type="button"
              >
                Calculate
              </button>
            </div>
          </div>
          <div
            class="flex-auto px-4 py-10 pt-0 lg:px-10"
          >
            <form>
              <hr
                class="mt-6 border-b-1 border-blueGray-300"
              />
              <h6
                class="mt-3 mb-6 text-sm font-bold text-gray-600 uppercase dark:text-white"
              >
                Current Price per byte-year: 
                <span
                  class="font-extrabold text-gray-700 dark:text-blue-500"
                >
                  0.00000348 SOL
                </span>
              </h6>
            </form>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
